src/eduid/webapp/idp/tests/test_SSO.py

```diff
@@ -240,6 +249,29 @@ def _get_login_response_authn(
 
             return login_next_step(ticket, sso_session_1)
 
+    @staticmethod
+    def _check_login_response_authn(
+        authn_result: NextResult,
+        message: IdPMsg,
+        expect_success: bool = True,
+        accr: Optional[EduidAuthnContextClass] = None,
+        assurance_profile: Optional[list[SwamidAssurance]] = None,
+        expect_error: Optional[bool] = False,
+    ):
+        assert authn_result.message == message, f"Message: {authn_result.message}, Expected: {message}"
+        if expect_success:
+            assert authn_result.authn_info
+            assert (
+                authn_result.authn_info.class_ref == accr
+            ), f"class_ref: {authn_result.authn_info.class_ref}, Expected: {accr}"
+            if assurance_profile is not None:
+                assert authn_result.authn_info.authn_attributes["eduPersonAssurance"] == [
+                    item.value for item in assurance_profile
+                ], "assurance profile does not match"
+        else:
+            assert authn_result.authn_info is None, f"authn_info: {authn_result.authn_info}, Expected: None"
+            assert authn_result.error is expect_error, f"error: {authn_result.error}, Expected: {expect_error}"
+
     # ------------------------------------------------------------------------
 
     def test__get_login_response_1(self):
```

Comment:
This file seems to be the only one where tests are named test__* instead of test_* (with double underscores), should they be renamed for consistency?